packages/front-end/components/License/ShowLicenseInfo.tsx

```diff
@@ -35,7 +36,7 @@ const ShowLicenseInfo: FC<{
 
   // TODO: Remove this once we have migrated all organizations to use the license key
   const usesLicenseInfoOnModel =
-    !showUpgradeButton && !organization?.licenseKey;
+    isCloud() && !showUpgradeButton && !organization?.licenseKey;
```

Comment:
Not sure if I understand why we need the `isCloud()` check. Do self-hosted Growthbook orgs not use the license info from the `licenses` collection?

Reply:
Because self hosted orgs can also specify the license in an environmental variable rather than on the mongo organization object.   We want to show the license info for them too.  Only on cloud and only on old organizations still keeping the license data itself as properties on the organization, do we want to hide the license info, because it would not display correctly.  We will be migrating those organizations off in the coming week.